cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,58 @@
+import PageObject from '../PageObject';
+
+export class SettingsPageObject extends PageObject {
+    url = '/settings';
+
+    get usernameField() {
+       return cy.getByDataCy('update-username');
+    }
+
+    typeUsername(username) {
+        this.usernameField.clear().type(username);
+      }
+
+    assertProfilePage(username) {
+        return cy.url().should('contain', `/profile/${username}`);
+      }
+
+      get submitBtn() {
+        return cy.getByDataCy('submit-btn');
+      }
+
+      submitForm() {
+        this.submitBtn.click();
+      }
+
+    get bioField() {
+        return cy.getByDataCy('update-bio');
+      }
+
+      typeBio(bio) {
+        this.bioField.type(bio);
```

Comment:
Consider clearing the bio field before typing to ensure that any existing text is removed. This will make the behavior consistent with the typeUsername and typeEmail methods.